examples/displayio_layout_dial_simpletest.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2021 Kevin Matocha
+#
+# SPDX-License-Identifier: MIT
+#############################
+"""
+This is a basic demonstration of a Dial widget.
+"""
+
+import time
+import board
+import displayio
+import terminalio
+from adafruit_displayio_layout.widgets.dial import Dial
+
+# Fonts used for the Dial tick labels
+tick_font = terminalio.FONT
+
+display = board.DISPLAY  # create the display on the PyPortal or Clue (for example)
+# otherwise change this to setup the display
+# for display chip driver and pinout you have (e.g. ILI9341)
+
+
+# Define the minimum and maximum values for the dial
+minimum_value = 0
+maximum_value = 100
+
+# Create a Dial widget
+my_dial = Dial(
+    x=20,  # set x-position of the dial inside of my_group
+    y=20,  # set y-position of the dial inside of my_group
+    width=180,  # requested width of the di
```

Comment:
this comment got cut off I think. should be `diameter`

Reply:
Corrected, good catch.